frontend/src/src/components/CMList.svelte

```diff
@@ -158,9 +159,11 @@
           <CM bind:cm />
         {/each}
       {:else if activeTabTest === 'consultation'}
-        <p> Information about the house. </p>
-        // plot popupContent here
-
+        <div class="popupInformation">
+            <table>
+                {@html $popupInformation}
+            </table>
+        </div>
```

Comment:
by default you show all the information in the HTLM string, we want to show only few information, this part need modification